lib-es5/utils/index.js

```diff
@@ -791,6 +792,9 @@ function url(public_id) {
   if (version != null) {
     version = `v${version}`;
   }
+  if (exclude_version) {
+    version = '';
```

Comment:
@strausr , please try to simplify the entire version-related logic here... 
In any case, if exclude_version is true, the rest of the logic is irrelevant, so you can start with that check and put the rest in else clause.